src/Mesh/Grids.jl

```diff
@@ -433,4 +434,52 @@ end
 
 # }}}
 
+
+"""
+    grid_stretching(DFloat,
```

Comment:
I do not understand the purpose of this function. How is it any different than passing different ranges to the brick code from the start?